src/types/slider.ts

```diff
@@ -0,0 +1,16 @@
+export enum SliderOrientation {
+  HORIZONTAL = 'horizontal',
+  VERTICAL = 'vertical',
+}
+
+export interface CustomSliderProps {
```

Comment:
The interface seems cryptic.

Can you check https://baseweb.design/components/slider/ or other popular lib for slider component def?